### PR TITLE
Update doctorfatekentnelson.ms

### PR DESCRIPTION
**Pull request by ZargonKing**

### CODE REVIEW

**Comment by malonnnn** [comment]
line 12 to line 27 has plus signs that need to be deleted

File: CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms [hunk 1]
```diff
@@ -9,6 +9,20 @@
             'trail': '',
             'op': 'true',
             'speech': "I am Kent Nelson, a.k.a. Doctor Fate.",
-            'powers': array(),
-            'buffs': array()
-        )
+            'powers': array(
+                "/buff",
+                "/fly",
+                "/carryplayer",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms]: this is missing in perissions

File: CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms [hunk 2]
```diff
@@ -9,6 +9,20 @@
...
+                "/dismountplayer",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms]: this is missing in perissions

File: CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms [hunk 3]
```diff
@@ -9,6 +9,20 @@
...
+                "/jump 0-5",
+                "/run 0-6",
+                "/haste 0-3",
+                "/grow",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms]: this is missing in perissions

File: CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms [hunk 4]
```diff
@@ -9,6 +9,20 @@
...
+                "/shrink"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms]: this is missing in perissions

File: CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms [hunk 5]
```diff
@@ -9,6 +9,20 @@
...
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 20 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/doctorfatekentnelson.ms]: EVIDENCE REQUIRED: super strength

**Comment by ViceTOP** [comment]
You should add a beam to him :D

**Pull request closed without merging**